files/build_templates/default_users.json.j2

```diff
@@ -0,0 +1,7 @@
+{%- set users_dict = {
+                        username: {"password": password,
+                                   "expire": password_expire
+                                  }
```

Comment:
>                                   [](http://example.com/codeflow?start=0&length=34)

The indentation in this file is weird. Could you recheck?

Reply:
Fixed, thanks